requestor/stop.sh

```diff
@@ -0,0 +1,9 @@
+#   Stop the server gracefully.
+#   The best way to ensure graceful yapapi shutdown is to send SIGTERM (15) 
+#   to the main gunicorn process and this is not obvious at all, thus this script.
+#   (check e.g. https://github.com/benoitc/gunicorn/issues/2604)
+
+#   NOTE: we can't use the standard `docker stop ...` because gunicorn is not the
+#   "main" container process - the main process is yagna_and_server_init.sh.
+GUNICORN_MASTER_PID=$(ps -aux | grep gunicorn | tr -s ' ' | cut -d ' ' -f2 | sort -n | head -1)
+kill -15 $GUNICORN_MASTER_PID
```

Comment:
It would be nicer to trap `SIGTERM` and stop the server inside `yagna_and_server_init.sh`. Check out [this solution](https://unix.stackexchange.com/a/146770).

Reply:
9d5414b

No trap was required (although trapping-based solution also works). 
Dockerfile ENTRYPOINT change was necessary.